test/e2e-ebpf/e2e_ebpf_suite_test.go

```diff
@@ -0,0 +1,80 @@
+package e2e_ebpf
+
+import (
+	"context"
+	_ "embed"
```

Comment:
Package `embed` shoud be removed from the import list.

Reply:
deleted